projects/storefrontlib/src/cms-components/configurator/commons/config-group-menu/config-group-menu.component.ts

```diff
@@ -25,7 +24,6 @@ export class ConfigGroupMenuComponent implements OnInit {
 
   iconTypes = ICON_TYPE;
   constructor(
-    private routingService: RoutingService,
```

Comment:
The standard set of changes :)
- return types of methods
- as there are no inputs, remove ngOnInit and use inline initialization
- as there are no inputs, remove `changeDetection: ChangeDetectionStrategy.OnPush`
- remove `public` modifiers
- favour `protected` over `private`

Reply:
Also, we can tackle the explicit subscriptions